src/emqx_alarm.erl

```diff
@@ -375,6 +375,8 @@ normalize_message(high_cpu_usage, #{usage := Usage}) ->
     list_to_binary(io_lib:format("~p% cpu usage", [Usage]));
 normalize_message(too_many_processes, #{usage := Usage}) ->
     list_to_binary(io_lib:format("~p% process usage", [Usage]));
+normalize_message(license_usage, #{high_watermark := High}) ->
+    iolist_to_binary(["License: the number of connections exceeds ", High, "%"]);
```

Comment:
I already transform the percentage by：
```erlang
emqx_alarm:activate(license_quota,
                #{high_watermark => float_to_binary(High * 100, [{decimals, 0}])});
```

Reply:
Is there any reason to transform the data before passing it to this function? It doesn't look in line with the other clauses that use `io_lib:format()`

Reply:
I guess it's because this map is used for both log and alarm, so there is no need to format twice ?

nit: a format function like fmt_percent would be good maybe.